source/refills/tooltip.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href="#0" class="tooltipped" aria-label="Tooltip description">
```

Comment:
We should test this, but I believe `aria-label` _overrides_ whatever content was mapped to the label by default (I believe in this case, the content within the `a` element).